lib/jekyll/polyglot/patches/jekyll/site.rb

```diff
@@ -112,12 +113,28 @@ def process_documents(docs)
       rel_regex = relative_url_regex
       abs_regex = absolute_url_regex(url)
       docs.each do |doc|
+        cleanLang(doc)
         relativize_urls(doc, rel_regex)
         if url
         then relativize_absolute_urls(doc, abs_regex, url)
         end
       end
     end
+      
+     # removes defined languages in .lang format (.es .fr) from url and document
+     def cleanLang(doc)
+      regex = ''
+      @languages.each do |lang|
+        regex += "\\.#{lang}|"
+      end
+      regex.chomp! '|'
+      unless doc.url.nil?
+        doc.url.gsub!(%r{#{regex}}, "")
+      end
+      unless doc.output.nil?
+        doc.output.gsub!(%r{#{regex}}, "")
+      end
+    end
```

Comment:
I would like just a few example tests written for this function `cleanLang` (shouldn't be too hard, just stick 'em in [here](https://github.com/untra/polyglot/blob/master/spec/tests/lib/jekyll/polyglot/patches/jekyll/site_spec.rb) on the branch) , and I'll take a look at why ruby head is failing on travis.

One suggestion for writing ruby functions: the last run statement is the return of the function. So if either `.gsub!` call runs, cleanLang is returning a string (see [gsub! ruby-doc](https://ruby-doc.org/core-2.4.2/String.html#method-i-gsub-21)) , but if both doc.url and doc.output are nil, then the last valid statement `regex.chomp! '|'` is returned. This means cleanLang is returning a regex, a different type.

to assist with writing the test, please just return the mutated `doc` at the end of the function 👍 
```suggestion
      doc
    end
```